proprietary/Components/src/denhaag/descriptionlist.tokens.json

```diff
@@ -0,0 +1,123 @@
+{
+  "denhaag": {
+    "descriptionlist": {
+      "border-color": {
+        "value": "{denhaag.color.grey.2}"
+      },
+      "color": {
+        "value": "{denhaag.color.grey.4}"
+      },
+      "font": {
+        "family": {
+          "value": "{denhaag.typography.sans-serif.font-family}"
+        },
+        "size": {
+          "value": "1.125rem"
+        },
+        "weight": {
+          "value": 400
+        }
+      },
+      "line-height": {
+        "value": 1.5
+      },
+      "padding": {
+        "inline": {
+          "end": {
+            "value": "{denhaag.space.inline.xs}"
+          },
+          "start": {
+            "value": "{denhaag.space.inline.xs}"
+          }
+        }
+      },
+      "caption": {
+        "color": {
+          "value": "{denhaag.color.black}"
+        },
+        "font": {
+          "family": {
+            "value": "{denhaag.typography.sans-serif-alternate.font-family}"
+          },
+          "size": {
+            "value": "{denhaag.typography.scale.xl.font-size}"
+          },
+          "weight": {
+            "value": 700
+          }
+        },
+        "line-height": {
+          "value": 1.3
+        },
+        "margin": {
+          "block": {
+            "value": "{denhaag.space.inline.xl}"
+          }
+        }
+      },
+      "detail": {
+        "padding": {
+          "block": {
+            "end": {
+              "value": "0.75rem"
+            },
+            "start": {
+              "value": "{denhaag.space.inline.2xs}"
+            }
+          }
+        }
+      },
+      "title": {
+        "font-weight": {
+          "value": 700
```

Comment:
I think it might be worthwhile to extract this font-weight to a typography token `denhaag.typography.weight.{regular|bold}`. It's reoccuring pretty often... in other components as well.